semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/HttpIncubatingAttributes.java

```diff
@@ -309,23 +277,18 @@ public static final class HttpConnectionStateValues {
     private HttpConnectionStateValues() {}
   }
 
-  /**
-   * Values for {@link #HTTP_FLAVOR}.
-   *
-   * @deprecated Deprecated, use `network.protocol.name` instead.
-   */
-  @Deprecated
```

Comment:
Something's fishy. This is still deprecated.

https://github.com/open-telemetry/semantic-conventions/blob/5eb37f706894718493af7a668971c651acb80812/docs/attributes-registry/http.md#deprecated-http-attributes

Reply:
I removed deprecated annotations from enum message definitions, adding them back in.

Reply:
Fixed